environment.devenv.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
     - cherrypy
     - hypothesis ==3.11.0
     - pytest-mock
-    - pytest-qt
+    - pytest-qt==3.3.0
```

Comment:
Without pin it was latest pytest-qt (4.0.1) which does not support our Qt version:

```ERROR: PyQt < 5.11 is not supported by pytest-qt, use 5.11+ instead.``` 

But this pin should be removed once we upgrade Qt to 5.12.

Reply:
Yeah I noticed but didn't comment, but I already updated to the latest pytest-qt on my Qt 5.12 branch. 👍